lib_camera/src/sensors/sensor_imx219.hpp

```diff
@@ -156,6 +156,13 @@ class IMX219 : public SensorBase {
      */
     int set_test_pattern(uint16_t pattern);
 
+    /**
+     * @brief Check if sensor is connected
+     *
+     * @returns           0 if succeeded, -1 if failed
+     */
+    int check_sensor_is_connected();
```

Comment:
```suggestion
    int check_sensor_is_connected() override;
```

Best to use `override` when overriding virtual methods to opt in to extra compiler checks (I think)

Reply:
Interesting topic, I checked, our compiler catches anyway as error, this lib is -Werror, so no need for now. 
(btw some compilers araise a warning here indeed, so is somth to have in mind)